### PR TITLE
Added an API to get the traceback of a failed computation

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1434107

### CODE REVIEW

**Comment by daniviga** [comment]
LGTM

File: openquake/server/v1/calc_urls.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
     url(r'^list$', 'calc'),
     url(r'^(\d+)$', 'calc_info'),
     url(r'^(\d+)/results$', 'calc_results'),
+    url(r'^(\d+)/traceback$', 'get_traceback'),
     url(r'^(\d+)/log/size$', 'get_log_size'),
     url(r'^(\d+)/log/(\d*):(\d*)$', 'get_log_slice'),
     url(r'^result/(\d+)$', 'get_result'),
```
Comment (nastasi-oq) [inline openquake/server/v1/calc_urls.py]: we must plan a more consistent API with result after the ID

File: openquake/server/v1/calc_urls.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
     url(r'^list$', 'calc'),
     url(r'^(\d+)$', 'calc_info'),
     url(r'^(\d+)/results$', 'calc_results'),
+    url(r'^(\d+)/traceback$', 'get_traceback'),
     url(r'^(\d+)/log/size$', 'get_log_size'),
```
Comment (nastasi-oq) [inline openquake/server/v1/calc_urls.py]: I think ```logsize``` is more correct than ```log/size```

**Comment by nastasi-oq** [comment]
LGTM

**Pull request merged**